pkg/logger/logger.go

```diff
@@ -14,41 +13,45 @@ var (
 	le   *log.Entry
 )
 
-func AddLogger(json bool) *log.Entry {
+func AddLogger() *log.Entry {
 	once.Do(func() {
-		logger := newLogger(json)
+		logger := newLogger()
 		le = log.NewEntry(logger)
 	})
 
 	return le
 }
 
-func newLogger(json bool) *log.Logger {
+func newLogger() *log.Logger {
 	logger := log.New()
+	logger.SetFormatter(customFormatter())
+	logger.SetReportCaller(true)
+	logger.SetLevel(log.DebugLevel)
```

Comment:
Curious thought: and by no means this is blocking merging of this PR. 
is there a way, we can set the LogLevel var to 'debug' or 'info' in the tfvars... and then modify the level here - based on the same. 
that way, we can add more debug logging - and not have it logged if it was info level during test run.

Reply:
like have a var on tf.vars ?

level_log = {desired level}  ? 

if yes, yes!

lol

but i would not recommend tie this up with any config env, especially tf